src/validator/GetSubgraphValidator.h

```diff
@@ -34,9 +34,10 @@ class GetSubgraphValidator final : public TraversalValidator {
     Status zeroStep(PlanNode* depend, const std::string& inputVar);
 
 private:
-    std::unordered_set<EdgeType>                edgeTypes_;
```

Comment:
Revert it.

Reply:
This is auto-formated by clang formatter.